package.json

```diff
@@ -31,38 +36,38 @@
   },
   "homepage": "https://github.com/A11yance/axobject-query#readme",
   "devDependencies": {
-    "@babel/cli": "^7.19.3",
-    "@babel/core": "^7.19.6",
-    "@babel/eslint-parser": "^7.19.1",
-    "@babel/preset-env": "^7.19.4",
-    "@babel/preset-flow": "^7.18.6",
-    "babel-jest": "^29.2.1",
+    "@babel/cli": "^7.24.7",
+    "@babel/core": "^7.24.7",
+    "@babel/eslint-parser": "^7.24.7",
+    "@babel/plugin-transform-react-jsx": "^7.24.7",
+    "@babel/preset-env": "^7.24.7",
+    "@babel/preset-flow": "^7.24.7",
+    "@babel/register": "^7.24.6",
+    "babel-plugin-module-resolver": "^5.0.2",
+    "encoding": "^0.1.13",
```

Comment:
why this dependency? it's not used in this PR

Reply:
It needs to be present since it’s a peer dep of something, and older npm doesn’t install peers by default.